src/procedures/procedures.11tydata.yaml

```diff
@@ -367,6 +367,152 @@ proceduresInstallingHAOS:
+        </ul>
+    - title: Open Raspberry Pi Imager      
+      image: yellow_rpi-imager_select_device_cm5.png
+      content: |
+        <ul>
+          <li>Open Raspberry Pi Imager.</li>
+          <li>Under <b>Raspberry Pi Device</b>, select </b>Raspberry Pi 5</b>.</li>     
+        </ul>
+    - title: Choosing the operating system
+      image: yellow_rpi-imager_cm5_module_image.webp
+      content: |
+        <ul>
+          <li>Select <b>Choose OS</b></li>
+          <li>Choose <b>Other specific-purpose OS</b>.</li>
+          <li>Choose <b>Home assistants and home automation</b>.</li>
+          <li>Choose <b>Home Assistant</b>.</li>
+          <li>Choose <b>Home Assistant OS 14.1 (Yellow)</b>.</li>
+        </ul>
+    - title: Choosing storage
+      image: rpi_choose_storage_cm5.png
+      content: |
+        <ul>
+          <li>Select <b>Choose storage</b> and from the dropdown menu, select your Raspberry Pi module.</li>
+        </ul>
+    - title: Writing the installer onto the drive
+      image: rpi_choose_next.png
+      content: |
+        <ol>
+          <li>Select the <b>Next</b> button to start the process.</li>
+          <li>Wait for the Home&nbsp;Assistant OS Installer to be written to the internal eMMC storage on your module.</li>
+          <li>Follow the wizard.</li>
+          <li>When the steps are complete, close the rpi-installer.</li>
+        </ol>
+    - title: Unplug and set to UART
+      image: yellow_jp1_to_uart.png
+      content: |
+        <ol>
+          <li>Unplug the USB cable.</li>
+          <li>Unplug the power cable.</li>
+          <li>Set the JP1 jumper to UART.</li>
+        </ol>
+proceduresPowerUpCM5:
+  title: Powering up and getting started
+  steps:
+    - title: Power up and watch the LEDs
+      image: yellow_heartbeat_yellow_led.webp
+      content: |
+        <ul>
+          <li>Connect Ethernet and power.</li>
+          <li>Start up may take a while. Once te yellow LED blinks in a heartbeat pattern, the system is running.</li>          
+        </ul>
+    - title: Getting started with Home Assistant
+      image: HA-get-started-02.png
+      content: |
+        <ul>
+          <li><b>Install the app</b></br>To access Home Assistant from your mobile device, use the QR code to locate the app in the app store.</li>
+          <li><b>Select your Home Assistant Server</b>: 
+            <ul>
+              <li><b>Mobile</b>: Confirm the IP address detected by the app. E.g. <a href="http://192.168.1.196:8123" target="_blank">http://192.168.1.196:8123</a></li></li>
+              <li><b>Desktop</b>: Visit <a href="http://homeassistant.local:8123" target="_blank">http://homeassistant.local:8123</a> to access the Home Assistant frontend.</li></li>
+            </ul>
+          <li>The Home Assistant frontend guides you through the initial onboarding.</li>
```

Comment:
I guess we should mention backup restore here? :thinking:

Reply:
ok, done